packages/application-shell/package.json

```diff
@@ -94,6 +87,22 @@
     "uuid": "3.3.2"
   },
   "devDependencies": {
+    "@commercetools-frontend/ui-kit": "^2.0.0-rc.0",
+    "react": "16.5.2",
+    "react-apollo": "2.2.4",
+    "react-dom": "16.5.2",
+    "react-intl": "2.7.1",
+    "react-router": "4.3.1",
+    "react-router-dom": "4.3.1",
     "wait-for-observables": "1.0.3"
+  },
+  "peerDependencies": {
+    "@commercetools-frontend/ui-kit": ">=2 <3 || >=2.0.0-rc.1 <3",
+    "react": ">=16",
```

Comment:
Might want to have a guard against the biggest version too. This is like claiming: I work with anything from the future which may be odd.

Reply:
Should we do that for every dep? Same then would apply to the ui-kit...

Reply:
Not sure. Personally I would only claim the library to be working until the next major release. Then, given that comes out, test with the next RC and depending on that update the range.

Reply:
Yeah, I'd do it for all deps - including ui-kit. We should also stop with the `rc` releases and just do regular ones. We're past the `rc` phase of the migration and back to normal development there anyways.

Reply:
Alright, updated